install_files/ansible-base/roles/common/tasks/apply_grsec_lock.yml

```diff
@@ -32,11 +32,18 @@
       delay=45
       state=started
   when: not grsec_lock.stat.exists
+  # Local wait action, doesn't require sudo privileges.
```

Comment:
If you didn't set `sudo: false`, what would happen? Would that mean you would need to run the playbook as root? Would Ansible error out when you started the playbook run, or only fail once it got to this task? This isn't important, I'm just curious.